src/RealTime/Localization/Translations/fr.xml

```diff
@@ -8,12 +8,12 @@
   <translation id="ResetToDefaults" value="Tout réinitialiser aux valeurs par défaut" />
 
   <translation id="1General" value="Général" />
-  <translation id="0Time" value="Time options" />
-  <translation id="1Other" value="Other options" />
-  <translation id="WakeupHour" value="City wakes up" />
-  <translation id="WakeupHourTooltip" value="The time when most citizens start waking up. The budgets and the time speed switch to the 'day' mode!" />
-  <translation id="GoToSleepUpHour" value="City goes to sleep" />
-  <translation id="GoToSleepUpHourTooltip" value="The time when most citizens start going to sleep. The budgets and the time speed switch to the 'night' mode!" />
+  <translation id="0Time" value="Options de Time" />
+  <translation id="1Other" value="Autres options" />
+  <translation id="WakeupHour" value="La ville se lève à" />
+  <translation id="WakeupHourTooltip" value="Le moment où la plupart des citoyens commencent à se réveiller. Les budgets et la vitesse temporelle passent en mode "jour"!" />
```

Comment:
I shouldn't have approved this change because there's an XML breaking error at this line.
I've already fixed it in my repository. Please pay attention next time to not break the format!